http.md

```diff
@@ -0,0 +1,301 @@
+
+```bash
+curl http://localhost:8899 -X POST -H "Content-Type: application/json" -d ' { "jsonrpc": "2.0", "id": 1, "method": "getBundle", "params": [ "bundle_id", { "commitment": "finalized" } ] }'
+```
+
+#### Default
+
+If commitment configuration is not provided, the node will default to `"finalized"` commitment
+
+Only methods that query bank state accept the commitment parameter. They are indicated in the API Reference below.
+
+#### RpcResponse Structure
+
+Many methods that take a commitment parameter return an RpcResponse JSON object comprised of two parts:
+
+- `context` : An RpcResponseContext JSON structure including a `slot` field at which the operation was evaluated.
+- `value` : The value returned by the operation itself.
+
+## Health Check
+
+TODO
+
+## JSON RPC API Reference
+
+## getTipAccounts
+
+Returns the tip accounts for tip payment for the bundles.
+
+### Parameters
+
+None
+
+### Result
+
+The result field will be a JSON object with the following fields:
+
+- `result`: `<array>` - Tip accounts as a list of `strings`
+
+### Code sample
+
+#### Request
+
+```bash
+curl http://localhost:8899 -X POST -H "Content-Type: application/json" -d '
+  {
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "getTipAccounts"
+  }
+'
+```
+
+#### Response
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": [
+        "9n3d1K5YD2vECAbRFhFFGYNNjiXtHXJWn9F31t89vsAV",
+        "aTtUk2DHgLhKZRDjePq6eiHRKC1XXFMBiSUfQ2JNDbN",
+        "B1mrQSpdeMU9gCvkJ6VsXVVoYjRGkNA7TtjMyqxrhecH",
+        "9ttgPBBhRYFuQccdR1DSnb7hydsWANoDsV3P9kaGMCEh",
+        "4xgEmT58RwTNsF5xm2RMYCnR1EVukdK8a1i2qFjnJFu3",
+        "EoW3SUQap7ZeynXQ2QJ847aerhxbPVr843uMeTfc9dxM",
+        "E2eSqe33tuhAHKTrwky5uEjaVqnb2T9ns6nHHUrN8588",
+        "ARTtviJkLLt6cHGQDydfo1Wyk6M4VGZdKZ2ZhdnJL336"
+    ],
+    "id": 1
+}
+```
+
+## sendBundle
+
+Submits a bundled list of signed transaction to the cluster for processing. The transactions will be processed in order and will be atomic meaning if at least one of the transactions fail, the bundle won’t be processed. It’s all or nothing. This method does not alter the transaction in any way; it relays the transaction created by clients to the node as-is. If the node's rpc service accepts the bundle, this method immediately succeeds, without waiting for any confirmations. A successful response from this method does not guarantee the transaction is processed or confirmed by the cluster.
+
+### Parameters
+
+None
+
+### Result
+
+The result field will be a JSON object with the following fields:
+
+- `result`: `string` - A bundle id which is the ash of all the first signatures of the transactions in the bundle, which is used to identify the bundle as a base-58/64 encoded `string`
+
+### Code sample
+
+#### Request
+
+```bash
+curl http://localhost:8899 -X POST -H "Content-Type: application/json" -d '
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "sendBundle",
+    "params": [
```

Comment:
we might want to nest the bundles so you can attach other context to this like commitment, etc.

should look to see how other RPC endpoints handle nested list